### PR TITLE
Added DR script and updated Readme

### PR DESCRIPTION
**Pull request by t-mbifeld**

### CODE REVIEW

File: Disaster_Recovery/recovery_script.ps1 [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+            throw "The config file must be of type 'yaml'"
+        }
+        return $true 
+    })]
+    [System.IO.FileInfo]$config
+)
+
+# Start Region: Set user inputs
+
+$location = 'region'
```
Comment (julienstroheker) [inline Disaster_Recovery/recovery_script.ps1:20]: Can all of this be parameters as well ?

File: Disaster_Recovery/recovery_script.ps1 [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+    if (!$applianceId) {
+        throw "Appliance recreation has failed."
+    }
+    # TODO: Get every 30 seconds, timeout after 5 minutes
+    $applianceStatus = (az resource show --debug --ids "$applianceId" --query 'properties.status' -o tsv 2>> $logFile)
+    if ($applianceStatus -ne "Running") {
+        throw "Appliance is not in running state. Current state: $applianceStatus."
+    }
+
+    logH1 "Step 2/5: Arc resource bridge is back up and running"
```
Comment (julienstroheker) [inline Disaster_Recovery/recovery_script.ps1:205]: step 3 no ?

File: README.md [hunk 3]
```diff
@@ -1,14 +1,10 @@
...
+Azure Arc resource bridge (preview) is part of the core Azure Arc platform, and is designed to host other Azure Arc services.  This repository contains relevant resources for administering Azure Arc resource bridge (preview).
 
-As the maintainer of this project, please make a few updates:
+NOTE: Any resources here might still be in development and should only be used after consulting relevant documentation.
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+Learn more: Azure Arc resource bridge (preview) overview - Azure Arc | Microsoft Docs
```
Comment (julienstroheker) [inline README.md:8]: add a short description about the your DR folder what it contains

Comment (t-mbifeld) [reply README.md:14]: Added

File: Disaster_Recovery/recovery_script.ps1 (hunk 2 above)
Comment (t-mbifeld) [reply Disaster_Recovery/recovery_script.ps1]: It marks the completion of step 2, added a 'completed' tag at the end of every step to indicate this

File: Disaster_Recovery/recovery_script.ps1 (hunk 1 above)
Comment (t-mbifeld) [reply Disaster_Recovery/recovery_script.ps1]: Yes, fixed

**Review by julienstroheker — APPROVED** [review]
LGTM as v1

**Pull request merged**